### PR TITLE
(RE-6130) Update the description of the genconfig task

### PR DESCRIPTION
**Pull request by underscorgan**
This task should include updating genconfig2 and beaker-hostgenerator.

### CODE REVIEW

**Comment by underscorgan** [comment]
@waynr can you make sure this wording makes sense?

File: tasks/agent-tickets.rake [hunk 1]
```diff
@@ -68,7 +68,7 @@ you need to create a new ticket, block this ticket against that one, and add in
 DOC
 
   description[:genconfig] = <<-DOC
-Update the genconfig for beaker
+Update genconfig2 and beaker-hostgenerator for #{vars[:platform_tag]}.
```
Comment (waynr) [inline tasks/agent-tickets.rake]: I would use links to their respective github repos, https://github.com/puppetlabs/sqa-utils-gem and https://github.com/puppetlabs/beaker-hostgenerator. I would also mention that `sqa-utils-gem`/`genconfig2` can be skipped once QENG-3181 and QENG-2436 are resolved.

Comment (underscorgan) [inline tasks/agent-tickets.rake]: Great, thanks!

I wasn't sure where those actually were :)

**Comment by underscorgan** [comment]
@waynr This look good?

Sample ticket at https://jira1-test.ops.puppetlabs.net/browse/QENG-3296

**Comment by waynr** [comment]
@underscorgan :+1: from me!

**Pull request merged**